tools/CouchDB_MRI_Importer.php

```diff
@@ -20,7 +31,7 @@ class CouchDBMRIImporter
 
     // this is just in an instance variable to make
     // the code a little more readable.
-    private $Dictionary = array();
+    private $_Dictionary = array();
```

Comment:
would require to change all access to this variable ($this->Dictionary).
```suggestion
    private $Dictionary = array();
```

Reply:
I think that PHPCS flagged this as an error, something like "private variables must be prefixed with an underscore." I'll change the other uses of it in the code instead of reverting the change.

Reply:
Whether an underscore is required or required to not be there depends on if the standard you added for the tools directory is based on PEAR or PSR.. it's one of the annoying incompatibilities between the two

Reply:
Yes exactly. It's using our custom LORIS standard (derived from PEAR). So the underscores are necessary.